pkg/util/ip_reservation.go

```diff
@@ -110,11 +111,21 @@ func (ipAllocator *IPAllocator) GetUnreservedIPRange(cidr string, cloudInstances
 
 	for cidrIPRange := cloneIP(ip.Mask(ipnet.Mask)); ipnet.Contains(cidrIPRange) && err == nil; cidrIPRange, err = incrementIP(cidrIPRange, incrementStep29IPRange) {
```

Comment:
Can you change the name to reflect that cidrIPRange isn't actually a range, it's a single ip.